includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -229,21 +237,23 @@ function perflab_aao_get_disabled_autoloaded_options_table(): string {
 	);
 
 	$nonce = wp_create_nonce( 'perflab_aao_update_autoload' );
-	foreach ( $disabled_options_summary as $value ) {
+
+	foreach ( $disabled_options_summary as $option_name => $option_length ) {
 		$url            = esc_url_raw(
 			add_query_arg(
 				array(
 					'action'      => 'perflab_aao_update_autoload',
 					'_wpnonce'    => $nonce,
-					'option_name' => $value->option_name,
+					'option_name' => $option_name,
 					'autoload'    => 'true',
 				),
 				admin_url( 'site-health.php' )
 			)
 		);
 		$disable_button = sprintf( '<a class="button" href="%s">%s</a>', esc_url( $url ), esc_html__( 'Revert to Autoload', 'performance-lab' ) );
-		$html_table    .= sprintf( '<tr><td>%s</td><td>%s</td><td>%s</td></tr>', esc_html( $value->option_name ), size_format( $value->option_value_length, 2 ), $disable_button );
+		$html_table    .= sprintf( '<tr><td>%s</td><td>%s</td><td>%s</td></tr>', esc_html( $option_name ), size_format( $option_length, 2 ), $disable_button );
```

Comment:
With the PHPStan level updates, this line was now giving an error about `$option_name` potentially not being a string. In normal plugin operation this would always be a string. But since any other plugin can manipulate options as well, there's no guarantee that it will be an array of strings. So I've added this check in 3dd2f695379fde4006b84b1bfcd61c3e97c0dfca.